### PR TITLE
Clarify patch descriptions

### PR DESCRIPTION
**Pull request by KobeW50**

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/layout/playerbuttonbg/PlayerButtonBackgroundPatch.kt [hunk 1]
```diff
@@ -10,7 +10,7 @@ import org.w3c.dom.Element
 @Suppress("Deprecation", "unused")
 object ForcePlayerButtonBackgroundPatch : BaseResourcePatch(
     name = "Force hide player buttons background",
-    description = "Hide the dark background surrounding the video player controls at compile time.",
+    description = "Removes the dark background surrounding the video player controls at compile time.",
```
Comment (KobeW50) [inline src/main/kotlin/app/revanced/patches/youtube/layout/playerbuttonbg/PlayerButtonBackgroundPatch.kt]: Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the dark background surrounding the video player controls.",
```

File: src/main/kotlin/app/revanced/patches/youtube/layout/dimming/ShortsDimmingPatch.kt [hunk 2]
```diff
@@ -9,7 +9,7 @@ import app.revanced.util.patch.BaseResourcePatch
 @Suppress("unused")
 object ShortsDimmingPatch : BaseResourcePatch(
     name = "Hide shorts dimming",
-    description = "Hide the dimming effect on the top and bottom of Shorts video at compile time.",
+    description = "Removes the dimming effect at the top and bottom of Shorts videos at compile time.",
```
Comment (KobeW50) [inline src/main/kotlin/app/revanced/patches/youtube/layout/dimming/ShortsDimmingPatch.kt]: Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the dimming effect at the top and bottom of Shorts videos.",
```

File: src/main/kotlin/app/revanced/patches/youtube/layout/animated/AnimatedLikePatch.kt [hunk 3]
```diff
@@ -11,7 +11,7 @@ import app.revanced.util.patch.BaseResourcePatch
 @Suppress("unused")
 object AnimatedLikePatch : BaseResourcePatch(
     name = "Hide double tap to like animations",
-    description = "Hide the like animations when double tap the screen in the Shorts player at compile time.",
+    description = "Removes the like animations when double-tapping the screen in the Shorts player at compile time.",
```
Comment (KobeW50) [inline src/main/kotlin/app/revanced/patches/youtube/layout/animated/AnimatedLikePatch.kt]: Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the like animations that appear when double-tapping the screen in the Shorts player.",
```

File: src/main/kotlin/app/revanced/patches/youtube/layout/animated/AnimatedButtonBackgroundPatch.kt [hunk 4]
```diff
@@ -10,7 +10,7 @@ import app.revanced.util.patch.BaseResourcePatch
 @Suppress("unused")
 object AnimatedButtonBackgroundPatch : BaseResourcePatch(
     name = "Hide animated button background",
-    description = "Hides the background of the pause and play animated buttons in the Shorts player at compile time.",
+    description = "Removes the background of the animated pause and play buttons in the Shorts player at compile time.",
```
Comment (KobeW50) [inline src/main/kotlin/app/revanced/patches/youtube/layout/animated/AnimatedButtonBackgroundPatch.kt]: Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the background of the animated pause and play buttons in the Shorts player.",
```

**Comment by KobeW50** [comment]
Should the `Hide double tap overlay filter` patch description also say have the phrase "at compile time"?

**Comment by inotia00** [comment]
yeah it's better to do this to keep patch descriptions consistent

**Comment by KobeW50** [comment]
Added for the ytm patches as well

**Comment by inotia00** [comment]
Thanks

**Comment by KobeW50** [comment]
I forgot about `Shorts outline button`

**Comment by inotia00** [comment]
Don't worry, there are still a lot of things to clarify in the patch description and settings

Just merge it in next release cycle

**Comment by KobeW50** [comment]
> there are still a lot of things to clarify in the patch description and settings

What do you have in mind?

**Comment by inotia00** [comment]
The most representative example is the following issue:
https://github.com/inotia00/ReVanced_Extended/issues/2051

While refactoring the RVX patch, I lowercase all proper nouns like `Shorts`, `Clips`, `Thanks`, and `Library`
(both in the settings and in the patch description)

You will understand if you refer to the official ReVanced settings

in ReVanced:
Disable resuming Shorts player

in RVX:
Disable resuming shorts player

**Comment by KobeW50** [comment]
Should these proper nowns also be capitalized in patch names? For example, should `Change shorts repeat state` become `Change Shorts repeat state`?

(I'm guessing not bec of cli, but i just want to make sure)

**Comment by inotia00** [comment]
> Should these proper nowns also be capitalized in patch names? For example, should `Change shorts repeat state` become `Change Shorts repeat state`?

yeah we need to change the patch name too

**Comment by PROBLEMCHYLD** [comment]
Not sure where to post but I can't select anything on Android Oxygen 14 OnePlus 
Screenshot_2024-05-28-10-58-17-53_84d3000e3f4017145260f7618db1d683

**Comment by KobeW50** [comment]
@PROBLEMCHYLD 

Ask [here](https://t.me/revancifychat) or create a bug report in [this repository](https://github.com/decipher3114/Revancify).

**Comment by selfmusing** [comment]
> Not sure where to post but I can't select anything on Android Oxygen 14 OnePlus Screenshot_2024-05-28-10-58-17-53_84d3000e3f4017145260f7618db1d683

Bruh , just reduce the size of termux window using your fingers to zoom-in / zoom-out on that patch screen!!

**Comment by PROBLEMCHYLD** [comment]
Yeah it did work, but I shouldn't have to keep adjusting my terminal every time I run a new program. Yes, it's a bug. Thanks the work around helps. It only happened when I updated to the latest. Thanks again.

**Pull request merged**